docs/conf.py

```diff
@@ -3,7 +3,7 @@
 # Kuma documentation build configuration file, created by
 # sphinx-quickstart on Mon Aug  5 15:41:51 2013.
 #
-# This file is execfile()d with the current directory set to its containing dir.
+# This file is execfile'd with the current directory set to its containing dir.
```

Comment:
This refers to the the Python [execfile()](https://docs.python.org/2/library/functions.html#execfile) function. I don't think there's a grammatically correct way to say this. It should also be fixed [upstream](https://github.com/sphinx-doc/sphinx/blob/f3118e1cd6ecb428b0dc56300e2913fd86ccefa3/sphinx/templates/quickstart/conf.py_t#L9), not here.